test/issue-generator.js

```diff
@@ -20,8 +20,8 @@ function createIssueLinks(direction, parentKey, parentType, linkType) {
   };
 }
 
-export default function createTestIssue(key, type, color, parentKey, parentType, linkType) {
-  let baseIssue = {
+function createBaseIssue(key, type, color){
+    return {
```

Comment:
Do you have editorconfig set up for your editor? Your indentation isn't right, so I'm wondering if your editor isn't consuming the config set up for this project.